### DISCUSSION TITLE
[Script request] RustDesk

### DISCUSSION THREAD

**Discussion by CrazyWolf13** [community-scripts/ProxmoxVE#34] [Request script]
### Application Name

RustDesk

### Website

https://rustdesk.com/

### Description

 The Fast Open-Source Remote Access and Support Software

Switch from TeamViewer, AnyDesk, and Splashtop to RustDesk for a secure and reliable remote desktop experience with your own self-hosted servers. 

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.

**Comment by oerix** [comment]
+1 would be a great addition as an LXC not docker.

**Comment by YamiGhor** [comment]
This implementation would be awesome!

**Comment by MasterCATZ** [comment]
also would be interested , alongside twingate server

**Comment by ciprianmtc** [comment]
The script was just added. You can find the id_ed25519.pub for encrypted connections in /var/lib/rustdesk-server/ . Would be great if the proxmox script would copy in the lxc root folder this file.

**Reply by tremor021** [reply]
> Would be great if the proxmox script would copy in the root folder this file.

why is that?

**Reply by ciprianmtc** [reply]
> > Would be great if the proxmox script would copy in the root folder this file.
> 
> why is that?

I mean in the lxc root folder. For ease of use ofc. Doesn't say anything in the rustdesk documentation where this file would be located after install.

**Reply by tremor021** [reply]
![image](https://github.com/user-attachments/assets/875a27df-435c-492c-b588-9116be9a9282)

it does tell you

if you look at the service:

`WorkingDirectory=/var/lib/rustdesk-server/`

**Comment by MickLesk** [comment]
closed as merged

**Reply by MasterCATZ** [reply]
Having issues getting android phone to connect any ideas how to fix key mismatched its not even asking for a password

**Reply by MickLesk** [reply]
Do an seperate issue or discussion, this is just the Request
